source/tutorials/multi-boot/dual-boot-win.rst

```diff
@@ -92,31 +89,33 @@ For this method, we shrink the Windows 10 OS partition to make space for |CL|.
       follow :ref:`Advanced: Use systemd-boot to boot Windows 10 OS and 
       |CL| <advanced-systemd-boot>`.
 
-Method 2: Add another hard disk to your system where you installed |CL|
-***********************************************************************
+Method 2: Add another hard disk to your system where you installed Windows
+**************************************************************************
 
 #. Shutdown your system.
 
 #. Open your system and attach another hard drive.
 
 #. Power up your system.
 
-#. Follow the instructions in :ref:`bare-metal-install-desktop`, and launch
-   the |CL| installer.
+#. Follow one of these guides to install |CL|:
 
-#. In the :guilabel:`Required options` tab, choose :guilabel:`Select
-   Installation Media`.
+   * *Desktop* version: :ref:`bare-metal-install-desktop`
+   * *Server* version: :ref:`bare-metal-install-server`
 
-#. Within that menu, select :guilabel:`Destructive Installation`, and
-   select the new hard drive from the device list.
+   a. In the :guilabel:`Required options` tab, choose :guilabel:`Select
+      Installation Media`.
 
-   .. note::
+   #. Within that menu, select :guilabel:`Destructive Installation`, and
+      select the new hard drive from the device list.
+
+      .. note::
```

Comment:
Please change the word note to  `warning`. Syntax is the same. It's minor, but it helps protect the user and us.